SwiftSampleProject/SwiftyVokoderTests/SwiftyVokoderTests.swift

```diff
@@ -65,4 +94,18 @@ class SwiftyVokoderTests: XCTestCase {
         XCTAssertEqual(trainLine.name, "Blue Line")
         XCTAssertEqual(trainLine.identifier, "BLUE")
     }
+    
+    func testImportAllStops() {
+        let stops = Stop.vok_addWithArray(self.allStopDictionaries())
+        
+        XCTAssertEqual(stops.count, 300)
+        guard let stop = stops.filter({ stop in
+            return stop.identifier == testIdentifier
+        }).first else {
+            XCTFail("Could not find stop \(testIdentifier)")
+            return
+        }
+        
+        self.verifyGrandMilwaukeeStop(stop)
```

Comment:
I assume the stop you're looking for is the test identifier stop? Might be helpful to change the name of `testIdentifier` to `grandMilwaukeeIdentifier` or the name of this method to `verifyStopWithTestIdentifier` just to make that clearer.